modules/ROOT/pages/shared-resources.adoc

```diff
@@ -95,14 +98,39 @@ You can configure the <<create_domain_project,domain project>> by defining share
                http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
                http://www.mulesoft.org/schema/mule/ee/domain http://www.mulesoft.org/schema/mule/ee/domain/current/mule-domain-ee.xsd">
 
-    <!-- configure here resource to be shared within the domain -->
+    <!-- Example configuration for an HTTP Listener-->
+    <http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" >
+      <http:listener-connection host="0.0.0.0" port="8081" />
+    </http:listener-config>
 
 </domain:mule-domain>
 ----
+
+=== Configure Shared Resources in the Global Elements Tab
+
+You can configure shared resources by creating connector configurations in the *Global Elements* tab in Anypoint Studio. Before you can create connector configurations using Studio, you must add the module to your project.
```

Comment:
I'm a little confused by terminology: we say you have to first add a "module" to your "project" and then create "connector" configurations. Is adding a module the same as adding a connector? And are we adding to a "project," a "domain," or a "domain project"? 
This might not matter to your audience...unless that audience includes someone like me. lol